artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcdproxy-sa
+  namespace: kube-apiserver-storage
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcdproxy-crd-cr
+rules:
```

Comment:
I was not able to find how to define another rule, to allow update only on Status field. I looked into various places, including K8s RBAC documentation, but I was not able to find more about that. Do you have any pointers where can I look into?